oaknational/Oak-Web-Application
# Fix/tracking change not listed

## Description

Music year: 1991

- Updated Avo & removed ts-error-ignores
- Updated hubspot submit to send notListed if no schoolURN

## How to test

1. Go to https://deploy-preview-2726--oak-web-application.netlify.thenational.academy
2. Download a curriculum document
3. In the network tab payload, should display:
![Screenshot 2024-08-21 at 13 13 48](https://github.com/user-attachments/assets/7a4cd817-b1c8-424d-981e-ef3db76a18d0)

Otherwise there should be no changes to the tracking event
